### PR TITLE
fixing issues #16 and #17

### PR DESCRIPTION
**Pull request by aliostad**

### CODE REVIEW

File: TestResults.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <assemblies>
-  <assembly name="D:\projects\dev\contrib\Zipkin.Tracer\tests\Zipkin.Tracer.Tests\bin\Release\Zipkin.Tracer.Tests.DLL" environment="64-bit .NET 4.0.30319.42000 [collection-per-class, non-parallel]" test-framework="xUnit.net 2.0.0.2929" run-date="2016-07-05" run-time="16:28:54" config-file="D:\projects\dev\contrib\Zipkin.Tracer\packages\xunit.runner.console\tools\xunit.console.exe.Config" total="2" passed="2" failed="0" skipped="0" time="0.289" errors="0">
+  <assembly name="C:\Users\aliostad\GitHub\zipkin-csharp\tests\Zipkin.Tracer.Tests\bin\Release\Zipkin.Tracer.Tests.DLL" environment="64-bit .NET 4.0.30319.42000 [collection-per-class, non-parallel]" test-framework="xUnit.net 2.0.0.2929" run-date="2017-02-07" run-time="22:17:03" config-file="C:\Users\aliostad\GitHub\zipkin-csharp\packages\xunit.runner.console\tools\xunit.console.exe.Config" total="2" passed="2" failed="0" skipped="0" time="0.393" errors="0">
```
Comment (adriancole) [inline TestResults.xml]: hehe did you mean to remove this file?

File: .gitignore [hunk 2]
```diff
@@ -237,3 +237,5 @@ _Pvt_Extensions
 
 # FAKE - F# Make
 .fake/
+TestResults.xml
+TestResults.xml
```
Comment (adriancole) [inline .gitignore]: dupe

**Comment by aliostad** [comment]
Actually I am gonna send all in one big pull request? It seems I cannot create new PRs.

**Comment by aliostad** [comment]
`TestResults.xml` must be removed. Issue #17 
I did `git rm --cached` but didn't remove it anyway. I have removed it in my branch now.

**Comment by adriancole** [comment]
PS better to make a description that describes the change. That way git log is much more clean and doesn't require someone to login to github to understand a summary of the history.

first line of commit message is the description.. then a blank line, then which issues it fixes.
Note: github doesn't recognise the word "fixing" rather "fixes".

Ex. 
```
Attributes code to OpenZipkin and adds RestorePackages

This does X and Y because of blah. just a couple sentences if the
long rational is in issues already

Fixes #16
Fixes #17
```

**Comment by adriancole** [comment]
I'll sort out TestResults.xml on the way in

**Comment by adriancole** [comment]
fixed on the way in and closed via 61ae46a

if I were you, I'd delete my fork and recreate

... or reset your master 2 commits up, so that it doesn't conflict upstream
 ex `git reset HEAD~2 --hard` make sure if you do this, it is on your clone not this repo! :)

**Pull request closed without merging**